libs/MeshKernel/include/MeshKernel/Smoother.hpp

```diff
@@ -45,7 +48,7 @@ namespace meshkernel
         explicit Smoother(const Mesh2D& mesh);
 
         /// @brief Computes the smoother weights
-        void Compute();
+        void Compute(const std::vector<MeshNodeType>& nodeTypes);
```

Comment:
my initial intention was to keep compute method without arguments and pass what is needed in the constructor. it would be nice if we can have the same signature for the compute method for all algorithms